desc/magnetic_fields/_dommaschk.py

```diff
@@ -79,7 +81,7 @@ def __init__(
 
     @classmethod
     def fit_magnetic_field(  # noqa: C901 - FIXME - simplify
-        cls, field, coords, max_m, max_l, sym=False, verbose=1
+        cls, field, coords, max_m, max_l, sym=False, verbose=1, NFP=1
```

Comment:
kind of wondering now, would it be worth re-naming `m` to `n` here to be consistent with notation used elsewhere in the code?

Also do we want `max_m` (or whatever we call it) to be the actual max, or the max of the "effective" mode number m/NFP?

eg, elsewhere in the code if you set N=5 and NFP=3 you get modes `[0,5,10,15]`, so Nmax is really 15 although we would call that Nmax=5

Reply:
same for possibly renaming `l -> m`

Reply:
- the point of max m is taken, I can change that and make it clear in the docstring what this is
- I am slightly hesitant to change the notation only because this is the exact notation that matches the paper.